jcl/src/jdk.jcmd/share/classes/openj9/tools/attach/diagnostics/tools/Jstat.java

```diff
@@ -0,0 +1,177 @@
+/*[INCLUDE-IF Sidecar18-SE]*/
+/*******************************************************************************
+ * Copyright (c) 2019, 2019 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+
+package openj9.tools.attach.diagnostics.tools;
+
+import java.io.IOException;
+
+import com.ibm.tools.attach.target.AttachHandler;
+import com.ibm.tools.attach.target.IPC;
+import openj9.tools.attach.diagnostics.attacher.AttacherDiagnosticsProvider;
+import openj9.tools.attach.diagnostics.base.DiagnosticProperties;
+
+/**
+ * JStat A tool for statistics monitoring of another Java process
+ *
+ */
+@SuppressWarnings("nls")
+public class Jstat {
+
+	private static String vmid;
+	private static String statOption;
+
+	private static final String OPTION_CLASS = "-class";
+	private static final String OPTION_GC = "-gc";
+	private static final String[] OPTIONS = { OPTION_CLASS, OPTION_GC };
+
+	private static final String ERROR_AN_ARG_REQUIRED = "An argument is required";
+	private static final String ERROR_INVALID_ARG = "An invalid argument";
+	private static final String ERROR_INVALID_OPTION = "An invalid option";
+	private static final String ERROR_INVALID_VMID = "Can't monitor this utility JVM itself: ";
+	private static final String ERROR_NOT_EXIST_VMID = "No such process for vmid : ";
+	private static final String ERROR_OPTION_REQUIRED = "An <option> is required";
+	private static final String ERROR_VMID_REQUIRED = "A <vmid> is required";
+
+	private static String HELPTEXT = "jstat: obtain statistics information about a Java process%n" + " Usage:%n"
```

Comment:
Might as well put each line in the output on a different line in the code. Easier to read and maintain.